style/ruby/README.md

```diff
@@ -411,3 +424,239 @@ hash = { one: 1, two: 2, three: 3 }
 # também é bom, os símbolos não podem usar hífens
 hash = { "twenty" => 20, "twenty-one" => 21 }
 ```
+
+# Operadores e Fluxo de Controle
+
+## Não use `and` ou `or`
+`and` e `or` apesar de funcionar, evite usa-las. Elas não são um sinônimo dos operadores `&&` e `||`, então não devem ser usadas dessa maneira. Para expressões booleanas, use **sempre** `&&` e `||`.
```

Comment:
>Ao usar `and` ou `or`, a expressão à esquerda do operador é avaliada primeiro. Se a expressão à esquerda de `and` for falsa, a expressão à direita não será avaliada, pois o resultado geral será falso. Por outro lado, se a expressão à esquerda de `or` for verdadeira, a expressão à direita não será avaliada, pois o resultado geral será verdadeiro.
No entanto, deve-se ter cuidado ao usar `and` e `or` porque eles têm precedência mais baixa do que a maioria dos outros operadores em Ruby. Portanto, é recomendável usar `&&` e `||` em vez de `and` e `or`, especialmente em expressões complexas para evitar problemas de precedência.
https://rubystyle.guide/#and-or-flow